definitions/npm/parse_v1.9.x/flow_v0.37.x-/parse_v1.9.x.js

```diff
@@ -32,7 +32,9 @@ declare type $npm$parse$Parse = {
   Session: any, // TODO
   Storage: any, // TODO
   Query: Class<$npm$parse$ParseQuery>,
-  User: Class<$npm$parse$ParseUser>
+  User: Class<$npm$parse$ParseUser>,
+  masterKey: ?string,
```

Comment:
You don't need to set it, but it's always defined via a getter/setter and appears to default to null